pkgs/top-level/all-packages.nix

```diff
@@ -14305,6 +14305,8 @@ with pkgs;
 
   zplug = callPackage ../shells/zsh/zplug { };
 
+  antidote = callPackage ../shells/zsh/antidote { };
```

Comment:
This needs to be added at the right place. Even if this file is not sorted perfectly, adding `antidote` in the middle of two derivations starting with `z` is totally wrong.